cmd/common/client_builder.go

```diff
@@ -29,6 +31,33 @@ func (cb *ClientBuilder) KubeClientOrDie(name string) kubernetes.Interface {
 	return kubernetes.NewForConfigOrDie(rest.AddUserAgent(cb.config, name))
 }
 
+type clusterOperatorsClient struct {
+	innerConfigClient configclientset.Interface
+}
+
+func (c *clusterOperatorsClient) Create(co *configv1.ClusterOperator) (*configv1.ClusterOperator, error) {
+	return c.innerConfigClient.ConfigV1().ClusterOperators().Create(co)
+}
+
+func (c *clusterOperatorsClient) UpdateStatus(co *configv1.ClusterOperator) (*configv1.ClusterOperator, error) {
+	return c.innerConfigClient.ConfigV1().ClusterOperators().UpdateStatus(co)
+}
+
+func (c *clusterOperatorsClient) Get(name string, options metav1.GetOptions) (*configv1.ClusterOperator, error) {
+	return c.innerConfigClient.ConfigV1().ClusterOperators().Get(name, options)
+}
+
+type ClusterOperatorsClientInterface interface {
```

Comment:
This is important for us to be able to control what we use in the MCO itself and allow us to create meaningful mocks for proper unit tests.